README.md

```diff
@@ -250,10 +252,6 @@ One way to do that is with a GitHub badge.
 
 [![ColPrac: Contributor's Guide on Collaborative Practices for Community Packages](https://img.shields.io/badge/ColPrac-Contributor's%20Guide-blueviolet)](https://github.com/SciML/ColPrac)
 
-```markdown
-[![ColPrac: Contributor's Guide on Collaborative Practices for Community Packages](https://img.shields.io/badge/ColPrac-Contributor's%20Guide-blueviolet)](https://github.com/SciML/ColPrac)
-```
-
```

Comment:
this should not be deleted as it's not a duplicate of the above, it's actually quoting the text necessary to produce the badge above, to show how to create it 

it shows up like:
<img width="1054" alt="Screenshot 2022-10-22 at 00 51 38" src="https://user-images.githubusercontent.com/13448787/197306125-43af2f46-90b7-4d3e-811e-f97ac6acb286.png">

Reply:
Ah okay. I looked at that code and saw that it was the same code in the block above the markdown block, as well as in the markdown block itself. Other badges that I have seen just have the upper section, but not the markdown section. 

So you are saying that in order to create the badge you need the markdown section. If you can confirm that intuition, I can undo this change.

Reply:
The first block creates the badge, in order to show users the badge. The second block quotes that code, in order to show users the code which creates the badge, so that they may easily copy-and-paste it to get the badge.

Reply:
Okay very good. Yeah, I just reverted this change. So the markdown section is restored.